CONTRIBUTING.md

```diff
@@ -112,6 +112,11 @@ These should result in no errors.
 Make sure your name and email adress are in the AUTHORS file in your pull request.
 Once the pull request is approved it will be merged by one of the maintainers.
 
+### Merge conflicts
+Sometimes your contributions can give rise to [merge conflicts](https://www.atlassian.com/git/tutorials/using-branches/merge-conflicts)
```

Comment:
Since we're working on github, better to use its explanation about merge conflicts: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/about-merge-conflicts

Reply:
Check